biogrid/apps/biogrid-mvp/src/app/pages/simulate-page/simulate-page.tsx

```diff
@@ -34,17 +35,40 @@ export const SimulatePage = () => {
 
   const redirectToHome = () => {
     history.push('/');
-  }
+  };
+
+  const stateToGridItemRet = (state: any): GridItemRet[] => {
+    return state.nodes.map((node: any) => {
+      return {
+        gridItemName: node.value.gridItemName,
+        relativePosition: node.value.position ||
+          node.value.relativePosition || { x: 0, y: 0 },
+      };
+    });
+  };
+  const stateToGridItemLines = (state: any): GridItemRet[] => {
+    return state.edges.map((edge: any) => {
+      return {
+        fromItem: edge.v,
+        toItem: edge.w,
+      };
+    });
+  };
 
   useEffect(() => {
     getSimulationResults();
   }, []);
 
-
   return (
     <div className="simulation">
       {simulationResults && (
         <div className="results">
+          <SimulationBoard
+            height={simulationResults.townSize.height}
+            width={simulationResults.townSize.width}
+            items={stateToGridItemRet(simulationResults.states[0])}
+            lines={stateToGridItemLines(simulationResults.states[0])}
```

Comment:
Oh it just uses indices for now (the indexes represent each subsequent state). When animation is added, it will step through the indices, let me add that as a TODO and github issue